orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/CompleteTaskHandler.kt

```diff
@@ -49,7 +49,7 @@ class CompleteTaskHandler(
       } else {
         repository.storeStage(mergedContextStage)
 
-        if (message.status != SUCCEEDED) {
+        if (!setOf(SUCCEEDED, FAILED_CONTINUE).contains(message.status)) {
```

Comment:
`message.status !in setOf(SUCCEEDED, FAILED_CONTINUE)`

Reply:
done